content/learn/book/introduction/_index.md

```diff
@@ -25,16 +27,14 @@ Bevy has the following design goals:
 * **Fast**: App logic should run quickly, and when possible, in parallel
 * **Productive**: Changes should compile quickly ... waiting isn't fun
 
-Bevy is [built in the open by volunteers](/learn/book/contributing) using the [Rust programming language](https://www.rust-lang.org/). The code is free and open-source because we believe developers should fully own their tools. Games are a huge part of our culture and humanity is investing _millions_ of hours into the development of games. Why are we continuing to build up the ecosystems of closed-source monopolies that take cuts of our sales and deny us visibility into the tech we use daily? We believe that the developer community can do so much better.
+Bevy is [built in the open by volunteers](/learn/book/contributing) using free and open-source code because we believe developers should fully own their tools. Games are a huge part of our culture and humanity is investing _millions_ of hours into the development of games. Bevy is rewriting the game engine marketplace, producing a WebGPU rendering using the [Rust programming language](https://www.rust-lang.org/).
 
 For a more in-depth introduction, check out the [Introducing Bevy](/news/introducing-bevy/) blog post.
 
 <h2 class="warning">
     Stability Warning
 </h2>
 
-Bevy aims to be a general purpose game engine capable of handling any 2D or 3D workload. However Bevy is still in its infancy. <span class="warning">We are currently in the <i>prototyping</i> phase: important features are missing and APIs will change constantly.</span> If you are currently trying to pick an engine for your Next Big Project™, we recommend that you check out [Godot Engine](https://godotengine.org). It is currently much more feature-complete and stable. And it is also free, open-source, and [scriptable with Rust](https://github.com/GodotNativeTools/godot-rust)!
+Bevy aims to be a general purpose game engine capable of handling any 2D or 3D workload. <span class="warning">Bevy is currently in development: expect APIs changes with every update.</span> If you are currently trying to pick an engine for your game, there is also [Godot Engine](https://godotengine.org) for those looking for an open source game engine built without Rust, yet is [scriptable with Rust](https://github.com/GodotNativeTools/godot-rust).
```

Comment:
Just because something is being actively developed does not imply breaking API changes, because that entirely depends on the project's state and maturity. This particular change has lost conveying that Bevy is still not mature as a project and that features are missing still. That is a very important point to make clear to a potential user expecting something more fleshed out and stable, regardless of prose style.